tests/integration/api_packages_test.go

```diff
@@ -403,3 +426,25 @@ func TestPackageCleanup(t *testing.T) {
 		}
 	})
 }
+
+// This method removes all package related data (database and files)
+func removeAllPackageData(t *testing.T) {
+	assert.NoError(t, db.DeleteAllRecords("package"))
+	assert.NoError(t, db.DeleteAllRecords("package_version"))
+	assert.NoError(t, db.DeleteAllRecords("package_file"))
+	assert.NoError(t, db.DeleteAllRecords("package_property"))
+	assert.NoError(t, db.DeleteAllRecords("package_blob_upload"))
+	assert.NoError(t, db.DeleteAllRecords("package_cleanup_rule"))
+
+	duration, _ := time.ParseDuration("-10h")
+
+	pbs, err := packages_model.FindExpiredUnreferencedBlobs(db.DefaultContext, duration)
```

Comment:
doesn't this work on what is in the db which you have just emptied above?

Reply:
Oh I see this requires package_blob

Reply:
The package blobs table was not touched, so all existing blobs are unreferenced.